internal/commands/secrets/update_test.go

```diff
@@ -0,0 +1,173 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestUserUpdateHandler(t *testing.T) {
+	projectID := "projectID"
+	appID := "appID"
+	app := realm.App{
+		ID:          appID,
+		GroupID:     projectID,
+		ClientAppID: "test-abcde",
+		Name:        "test",
+	}
+
+	testLen := 7
+	secrets := make([]realm.Secret, testLen)
+	for i := 0; i < testLen; i++ {
+		secrets[i] = realm.Secret{
+			ID:   fmt.Sprintf("secretID%d", i),
+			Name: fmt.Sprintf("secretName%d", i),
+		}
+	}
+	successOutput := func(t *testing.T, output string) {
```

Comment:
we should just use this `assert.Equal` where we make the assert in the test

i think with my comment about removing the `ui.Print` when we fail to update the secret, that last test case here goes away.  if it doesn't for some reason, let's at least break it out into its own test func, we should try to avoid mixing success/failure test cases in a single test table like this (a benefit of this is it helps simplify the anonymous `struct{}`s we use for test cases)